src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__wrapper calendar__wrapper--start-day--mon calendar__wrapper--month-length--31">
+        <div class="calendar__wrapper-day"></div>
```

Comment:
Why do you need a wrapper at all?

Reply:
right! Better put the calendar in the wrapper. It helps center calendar block